types/inputmask/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "unified-signatures": false
```

Comment:
Please don't turn off lint rules on new packages. It really helps the maintainers and other contributors when code looks consistent throughout DefinitelyTyped.

Reply:
Thanks for the review. I have reenabled all lint rules and instead marked individual lines with `tslint:disable-line:unified-signatures` to allow individual documentation of each JQuery method.